src/app/child-dev-project/notes/note-details/note-details.component.html

```diff
@@ -193,5 +193,17 @@ <h1>{{ entity.date | date }}: {{ entity.subject }}</h1>
     >
     </app-entity-select>
 
+    <app-entity-select *ngIf="relatedEntitiesSchema?.additional"
+      class="form-section"
+      entityType="ChildSchoolRelation"
```

Comment:
```suggestion
      [entityType]="relatedEntitiesSchema?.additional"
```
I don't see an advantage of the hard-coding.